src/main/java/org/team1540/elmo/ejectors/BunnyEjectors.java

```diff
@@ -0,0 +1,54 @@
+package org.team1540.elmo.ejectors;
```

Comment:
This stuff should probably go in the commands folder, not its own top level folder

Reply:
done